brax/io/__init__.py

```diff
@@ -12,3 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+GFile = None
+try:
+  from tensorflow.io.gfile import GFile
+except:
+  pass
+
+class File:
+  def __init__ (self, fileName, mode='r'):
```

Comment:
To follow the code style in this rep, can you add another newline before each method?

Reply:
done